src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const CAR_RENT_COST = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const RENT_COST_ALL_TERM = days * CAR_RENT_COST;
```

Comment:
```suggestion
  const basicPrice = days * CAR_RENT_COST;
```
It's a calculated variable, so should be written in camelCase